Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift

```diff
@@ -28,652 +28,651 @@ func specializedAdd(_ lhs: BinaryTree<Double>, _ rhs: BinaryTree<Double>) -> Bin
     }
 }
 
-
 // ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 32)
 
 func specializedAdd(_ lhs: Point1, _ rhs: Point1) -> Point1 {
     return Point1(
-// ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 36)
+        // ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 36)
         _1: lhs._1 + rhs._1
-// ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 41)
+            // ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 41)
     )
 }
 
 // ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 32)
 
 func specializedAdd(_ lhs: Point2, _ rhs: Point2) -> Point2 {
     return Point2(
-// ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 36)
-        _1: lhs._1 + rhs._1
-// ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 38)
-        ,
-// ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 36)
+        // ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 36)
+        _1: lhs._1
+            + rhs._1// ###sourceLocation(file: "/usr/local/google/home/shabalin/swift-libs/swift-structural/Sources/StructuralBenchmarks/MyAdditiveBenchmarks.swift.gyb", line: 38)
```

Comment:
Did the version of gyb change?  This looks like a bug; souceLocation comments are supposed to be on their own lines.